Classes/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, name = 'Unknown', specialization: true)
+    super(name, age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly add `parent_permission` parameter to the initialize method with a default value of `true` and also add `parent_permission` to the super call so that we can have access to it from the parent class.

- [ ]  Note that the specialization is not supposed to have a default value, so please remove the default value you added to the specialization.